bin/check-hardware-fail.rb

```diff
@@ -29,9 +29,26 @@
 require 'sensu-plugin/check/cli'
 
 class CheckHardwareFail < Sensu::Plugin::Check::CLI
+  option :lines,
+         short: '-l NUMBER',
+         long: '--lines NUMBER',
+         proc: proc(&:to_i),
+         default: 0,
```

Comment:
whats the value of default of 0? `nil.to_i == 0`; is this just for clarity or does it have some other purpose eluding me?